src/twisted/internet/iocpreactor/abstract.py

```diff
@@ -188,48 +188,49 @@ def _resumeWriting(self):
 
 
     def _cbWrite(self, rc, numBytesWritten, evt):
-        if self._handleWrite(rc, numBytesWritten, evt):
+        if self._handleWrite(rc, numBytesWritten, evt) and \
+        self._writeScheduled is None:
```

Comment:
Please use parentheses for breaking long lines.
Also, either encapsulate the logic in a variable or add a comment to this conditional.

Ex
```
is_crash = self._someCheck() and self._otherThing
if is_crash:
    self.doDomething()
```
or

```
if self._someCheck() and self._otherThing:
    # Bungulu has crashed, so we clean before propagating the error.
    self.doDomething()
```

Reply:
I can certainly document this better.  I don't understand the parenthesis  suggestion.  Are you suggesting breaking down the one if statement to two?

Reply:
Breaking long lines

```
if self._handleWrite(rc, numBytesWritten, evt) and \
        self._writeScheduled is None:
    CODE_BLOCK_HERE
```

vs

```
if (
    self._handleWrite(rc, numBytesWritten, evt) and
    self._writeScheduled is None
        ):
    CODE_BLOCK_HERE
```

Reply:
I can make that change!  Adding a comment to explain the logic as well.